### ISSUE TITLE
Option to delete versions

### ISSUE THREAD

**Issue by rayzchen** [mindstorm38/portablemc#93]
How do I delete a version from `portablemc search -l` without manually doing it?

**Comment by mindstorm38** [comment]
You mean deleting the version's directory?

**Comment by rayzchen** [comment]
Is that all there is to it?

**Comment by mindstorm38** [comment]
Yes, this might be possible to also delete some assets and libraries specific to the version, but it's enough to delete a version.

**Comment by mindstorm38** [comment]
I've been thinking of this feature and this is definitely a missing feature of this launcher. Being able to manage installed versions, assets, logs can be really cool. Especially thinking of a log analyzer or assets cleanup, to free storage.

**Comment by mindstorm38** [comment]
I'm wondering which subcommand to add for this, something like `clean`. I will also add more things to the `search` subcommand, in order to list assets, libraries of a specific version. I'm also tempted to add a `log` subcommand, in order to analyze game logs.
